candidate-testing.js

```diff
@@ -1,35 +1,42 @@
-const input = require('readline-sync');
-
 // TODO 2: modify your quiz app to ask 5 questions //
 
 // TODO 1.1a: Define candidateName // 
-let candidateName;
+let candidateName = "";
 // TODO 1.2a: Define question, correctAnswer, and candidateAnswer //
-let question;
-let correctAnswer;
-let candidateAnswer;
+let question = "";
+let correctAnswer = "";
+let candidateAnswer = "";
 let questions;
 let correctAnswers;
 let candidateAnswers;
-
+let space = " ";
 
 function askForName() {
   // TODO 1.1b: Ask for candidate's name //
+const input = require('readline-sync');
+ candidateName = input.question("Please enter your name: " );
 
+console.log("Hello," + space + candidateName +"!");
 }
 
 function askQuestion() {
   // TODO 1.2b: Ask candidate the question and assign the response as candidateAnswer //
-
-
+const input = require('readline-sync');
```

Comment:
Just fyi, this was already declared on line 1 of the file. Just in case, you may want to delete this since it is a const. I wouldn't want this to give an error on you.